src/sql/workbench/services/accountManagement/browser/accountDialog.ts

```diff
@@ -209,6 +212,10 @@ export class AccountDialog extends Modal {
 
 	protected renderBody(container: HTMLElement) {
 		this._container = container;
+
+		// Setup loading spinner
```

Comment:
This comment isn't really adding any useful information (it's clear enough that new'ing up a loading spinner is going to set it up) - I'd suggest just removing this.